src/Razor/test/testassets/Directory.Build.props

```diff
@@ -2,21 +2,17 @@
   <Import Project="Before.Directory.Build.props" Condition="Exists('Before.Directory.Build.props')" />
 
   <PropertyGroup>
-    <!--
-      In the case that a user is building a sample directly the MicrosoftNetCompilersToolsetPackagerVersion will not be provided.
-      We'll fall back to whatever the current SDK provides in regards to Roslyn's Microsoft.Net.Compilers.Toolset.
-    -->
-    <BuildingTestAppsIndependently>false</BuildingTestAppsIndependently>
-    <BuildingTestAppsIndependently Condition="'$(MicrosoftNetCompilersToolsetPackageVersion)' == ''">true</BuildingTestAppsIndependently>
-
     <!-- Do not resolve Reference ItemGroup since it has a different semantic meaning in Razor test apps -->
     <EnableCustomReferenceResolution>false</EnableCustomReferenceResolution>
 
     <DefaultNetCoreTargetFramework>net6.0</DefaultNetCoreTargetFramework>
 
-    <RepoRoot Condition="'$(BuildingTestAppsIndependently)' == 'true' AND '$(RepoRoot)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, 'global.json'))\</RepoRoot>
+    <RepoRoot Condition=" '$(RepoRoot)' == '' ">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, 'global.json'))\</RepoRoot>
     <RazorSdkArtifactsDirectory>$(RepoRoot)artifacts\bin\Microsoft.NET.Sdk.Razor\</RazorSdkArtifactsDirectory>
     <BlazorWebAssemblyJSPath>$(MSBuildThisFileDirectory)blazor.webassembly.js</BlazorWebAssemblyJSPath>
+
+    <!-- Build these test projects against the incoming Microsoft.NETCore.App framework. -->
+    <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
```

Comment:
@dsplaisted and @wli3 given we're setting `%(DefaultRuntimeFrameworkVersion)` to be identical to `%(LatestRuntimeFrameworkVersion)` for the Microsoft.NETCore.App `@(KnownFrameworkReference)` item (in the generated Directory.Build.targets file imported later), does this setting (or making `$(RuntimeFrameworkVersion)` equal to that metadata) change anything at all❔

See https://github.com/dotnet/aspnetcore/pull/27473/files#diff-9fb67cbc0a7ba3b793455ec41e7f36e1d019d5152674d669551810b308510d37R26-R30

Reply:
@dsplaisted double check, i believe it is the same

Reply:
No, in this case setting `TargetLatestRuntimePatch` to true will get the version from the "latest" instead of the "default" metadata value, but if they are both the same then it won't affect everything.